src/Composition/ForceElements/Bushing.jl

```diff
@@ -1,29 +1,36 @@
 
 """
-force = Bushing(; obj1, obj2, stiffness, damping)
+    force = Bushing(; obj1, obj2,
+        nominalForce = Modia3D.ZeroVector3D,
+        stiffness    = Modia3D.ZeroVector3D,
+        damping      = Modia3D.ZeroVector3D )
 
 Return a `force` acting as bushing between `obj1::`[`Object3D`](@ref) and
-`obj2::`[`Object3D`](@ref). Vectors `stiffness` and `damping` define the
-stiffness resp. damping values in x, y and z direction of `obj1`. The
-orientation of `obj2` does not influence the resulting forces.
+`obj2::`[`Object3D`](@ref). Vectors `stiffness`, `damping` and
+`nominalForce` define the stiffness, damping and nominal force values in
+x, y and z direction of `obj1`. The orientation of `obj2` does not
+influence the resulting forces.
 """
 mutable struct Bushing <: Modia3D.AbstractForceElement
 
     obj1::Modia3D.AbstractObject3D
     obj2::Modia3D.AbstractObject3D
```

Comment:
Do you need type `AbstractObject3D` here, or is an `Object3D` sufficient?

Reply:
Please, do not use AbstractObject3D, We only have one Object3D struct type and therefore Object3D is neededhere

Reply:
`AbstractObject3D` is used in files: Bushing.jl, SpringDamperPtP.jl and object3Dmotion.jl

Reply:
Whenever an "abstract" Julia type is used in a declaration in a struct, then the type information is always resolved at run-time which gives very unefficient code. It is therefore of utmost importance to never use abstract types in a struct. If the type if not known, it must be provided as type parameter "mutable struct Test{T} ....", and then the concrete type has to be given when calling the constructor and at then the struct is compiled for this specific type which is again efficient.

Reply:
Good point, I am going to change this.

I copied it from `object3DMotion.jl/FreeMotion` where `AbstractObject3D` is necessary because `Object3D` is included afterwards.